sandbox/api/app.py

```diff
@@ -34,35 +61,131 @@ def get_related_persons() -> Union[dict, tuple]:
+        # Request with patient:identifier - but its not in a list of identifiers
+        return __generate_response(load_json_file(NOT_FOUND), 404)
+    elif identifier == "9000000033":
+        # Request with identifier for empty record
+        return __generate_response(load_json_file(EMPTY_RESPONSE))
+
+
+def __check_for_validate(
+    value: str,
+    identifier: str,
+    patient_identifier: str,
+    include: str,
+    basefile: str,
+    incfile: str,
+) -> Response:
+    """Checks for validate request responses for a given relationship record
+
+    Args:
+        value (str): NHS number of the relationship to look for
+        identifier (str): The identifier supplied to the request
+        patient_identifier (str): The patient:identifier supplied to the request
+        include (str): The include parameter supplied to the request
+        basefile (str): The file to return when record matches but does not request includeded data
+        incfile (str): The file to return when record matches and does request included data
+
+    Returns:
+        Response: Resultant response or None
+    """
+    if identifier and patient_identifier == value and include == INCLUDE_FLAG:
+        # Request with identifier, patient and _include=patient
+        return __generate_response(load_json_file(incfile))
+    elif identifier and patient_identifier == value:
+        # Request with identifier and patient
+        return __generate_response(load_json_file(basefile))
+
+
+def __check_for_list(
+    value: str, identifier: str, include: str, basefile: str, incfile: str
+) -> Response:
+    """Check for a list relationship response for a given NHS number
+
+    Args:
+        value (str): NHS number of the relationship to look for
+        identifier (str): The identifier supplied to the request
+        include (str): The include parameter supplied to the request
+        basefile (str): The file to return when record matches but does not request includeded data
+        incfile (str): The file to return when record matches and does request included data
+
+    Returns:
+        Response: Resultant response or None
+    """
+    if identifier == value and include == INCLUDE_FLAG:
+        # Request with identifier and _include=patient
+        return __generate_response(load_json_file(incfile))
+    elif identifier:
+        # Request with identifier
+        return __generate_response(load_json_file(basefile))
+
+
+def __generate_response(content: str, status: int = 200):
+    """Creates a response object with the supplied data and content type of "application/fhir+json"
+
+    Args:
+        content (json): The content for the response
+        status (int, optional): The status header of the response. Defaults to 200.
+
+    Returns:
+        Response: Resultant Response object based on input.
+    """
+    return Response(dumps(content), status=status, mimetype="application/fhir+json")
```

Comment:
Note `mimetype="application/fhir+json"` is also set by the response flow from apigee

Reply:
Oh I did not realise this.  I will leave it as is for now as it was not appearing when testing locally.